test/functional/mac/execute_script_test.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from appium.webdriver.common.mobileby import MobileBy
+from test.functional.mac.helper.test_helper import BaseTestCase
+from test.functional.test_helper import wait_for_element
+
+
+class TestExecuteScript(BaseTestCase):
+    def test_sending_custom_keys(self) -> None:
```

Comment:
it is good to have these tests, but they are not going to work in CI env, because the necessary accessibility permissions are missing there